### PR TITLE
B2b phase 1

### PR DESCRIPTION
**Pull request by tamigoodall**

### CODE REVIEW

File: articles/architecture-scenarios/_includes/_authentication/_introduction.md [hunk 1]
```diff
@@ -1,28 +1,38 @@
-In order to provide services to your users, you must be able to identify who those users are. This process is called User  Authentication. There are a number of ways to perform authentication of a user - via social media accounts, username & password, <dfn data-key="passwordless">passwordless</dfn> - and it's often recommended that you go beyond a first factor for authenticating the user by adding a second factor as well (a.k.a. Multi-factor Authentication).
+In order to provide services to your users, you must be able to identify who those users are. This process is called User  Authentication. There are a number of ways to perform authentication of a user - via social media accounts, username & password, passwordless - and it's often recommended that you go beyond a first factor for authenticating the user by adding a second factor as well (a.k.a. Multi-factor Authentication).
```
Comment (mostekcm) [inline articles/architecture-scenarios/_includes/_authentication/_introduction.md]: Do we want to have that data-key included?

File: articles/architecture-scenarios/_includes/_authentication/_introduction.md [hunk 2]
```diff
@@ -1,28 +1,38 @@
...
 
-For customer-facing applications, <dfn data-key="openid">[OpenID Connect (OIDC)](/protocols/oidc)</dfn> is the most frequently used industry standard protocol, and OIDC has first-class citizen support in Auth0. Auth0 provides support for various different approaches for integrating various different applications, so you'll want to see the section on [application integration](#application-integration) for the information you'll need to make an informed choice. 
+For customer facing applications, OpenID Connect ([OIDC](/protocols/oidc)) is the most frequently used industry standard protocol, and OIDC has first-class citizen support in Auth0. Auth0 provides support for various different approaches for integrating various different applications, so you'll want to see the section on [application integration](#application-integration) for the information you'll need to make an informed choice.
```
Comment (mostekcm) [inline articles/architecture-scenarios/_includes/_authentication/_introduction.md:36]: do we need the data-key included?

File: articles/architecture-scenarios/implementation/b2b/b2b-authentication.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+Once you start supporting enterprise connections for users, you must do some form of [Home Realm Discovery](#home-realm-discovery) so that you can determine which connection to send the user to for authentication.
+:::
+
+With enterprise connection support, user identities and credentials are managed by the identity provider of your customers' organization, as well as certain identity claims - which Auth0 will use to populate the user [profile](/architecture-scenarios/implementation/b2b/b2b-profile-mgmt).
+
+::: panel Best Practice
+"Bring your own identity" is a great feature to provide, but if you don't support this from day one, and sometimes even if you do, you may have an organization that wants to switch to their own IDP after already having used the application for a while.  You will need a way to [Link User Accounts](https://auth0.com/docs/link-accounts) (a.k.a. Account Linking) to provide an effective way of associating the new identity with the old database identity.
+:::
+
+## Machine-to-Machine (M2M) authentication
```
Comment (mostekcm) [inline articles/architecture-scenarios/implementation/b2b/b2b-authentication.md]:
```suggestion
```

File: articles/architecture-scenarios/implementation/b2b/b2b-authentication.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
```
Comment (mostekcm) [inline articles/architecture-scenarios/implementation/b2b/b2b-authentication.md]:
```suggestion
```

File: articles/architecture-scenarios/implementation/b2b/b2b-authentication.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+<%= include('../../_includes/_authentication/_m2m.md', { platform: 'b2b' }) %>
```
Comment (mostekcm) [inline articles/architecture-scenarios/implementation/b2b/b2b-authentication.md]:
```suggestion
```

File: articles/architecture-scenarios/implementation/b2b/b2b-authentication.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+
```
Comment (mostekcm) [inline articles/architecture-scenarios/implementation/b2b/b2b-authentication.md]:
```suggestion
```

File: articles/architecture-scenarios/implementation/b2c/b2c-authentication.md [hunk 7]
```diff
@@ -39,13 +39,17 @@ useCase:
 
 <%= include('../../_includes/_authentication/_social-authentication.md', { platform: 'b2c' }) %>
 
+## Machine-to-Machine (M2M) authentication
```
Comment (mostekcm) [inline articles/architecture-scenarios/implementation/b2c/b2c-authentication.md]:
```suggestion
```

File: articles/architecture-scenarios/implementation/b2c/b2c-authentication.md [hunk 8]
```diff
@@ -39,13 +39,17 @@ useCase:
...
+
```
Comment (mostekcm) [inline articles/architecture-scenarios/implementation/b2c/b2c-authentication.md]:
```suggestion
```

File: articles/architecture-scenarios/implementation/b2c/b2c-authentication.md [hunk 9]
```diff
@@ -39,13 +39,17 @@ useCase:
...
+<%= include('../../_includes/_authentication/_m2m.md', { platform: 'b2c' }) %>
```
Comment (mostekcm) [inline articles/architecture-scenarios/implementation/b2c/b2c-authentication.md]:
```suggestion
```

File: articles/architecture-scenarios/implementation/b2c/b2c-authentication.md [hunk 10]
```diff
@@ -39,13 +39,17 @@ useCase:
...
+
```
Comment (mostekcm) [inline articles/architecture-scenarios/implementation/b2c/b2c-authentication.md]:
```suggestion
```

File: articles/architecture-scenarios/_includes/_authentication/_m2m.md [hunk 11]
```diff
@@ -0,0 +1,14 @@
+There are many scenarios that require an application _without_ any user-interactive session to obtain an access token in order to call an API. In such scenarios you must authenticate the client instead of the user, and OAuth 2 provides the [client credentials](/flows/concepts/client-credentials) grant type to make this easy to achieve. Some common examples of where this is required include:
+* A cron job or other service that needs to communicate with your API (e.g. where a daily report needs to be generated and emailed it to an administrator).
+* A separate API the supports privileged access (e.g. the API is not exposed to users directly, but instead to a backend only).
+* In certain microservice architectures, where some API layers need to communicate to other API layers without a user involvement, or after a user token has expired.
+* A privileged API that may need to be called before a user has authenticated (i.e. from a rule or custom DB script in your Auth0 tenant)
+
```
Comment (mostekcm) [inline articles/architecture-scenarios/_includes/_authentication/_m2m.md]: This whole file needs to be removed...

Comment (mostekcm) [reply articles/architecture-scenarios/_includes/_authentication/_m2m.md]: it was replaced with _authorization/_m2m.md

File: articles/architecture-scenarios/_includes/_profile-mgmt/_introduction.md [hunk 12]
```diff
@@ -16,17 +16,24 @@ The Normalized User Profile is updated from the identity provider during login,
...
 :::
 
 Alternatively, the Auth0 Dashboard can be used to [manage aspects of a user’s profile](users/guides/manage-users-using-the-dashboard). Managing a user’s profile via the Auth0 Dashboard is more of an administrative provision and **should not** be used for self-serviced profile management in a production environment. However, the interface provided by the Dashboard can be extremely useful during development as it provides a quick and simple way of manipulating a user’s profile information.
+
+<% if (platform === "b2b") { %>
+* If you need to provide a way for your customers to have an administrator that can manage their own users when they are storing those credentials in your system, you can either build something yourself or use an Auth0 Extension. See [Admin Portal](#admin-portal) for more details.
```
Comment (mostekcm) [inline articles/architecture-scenarios/_includes/_profile-mgmt/_introduction.md]:
```suggestion
If you need to provide a way for your customers to have an administrator that can manage their own users when they are storing those credentials in your system, you can either build something yourself or use an Auth0 Extension. See [Admin Portal](#admin-portal) for more details. 
```

File: articles/architecture-scenarios/_includes/_provisioning/_introduction.md [hunk 13]
```diff
@@ -6,10 +6,23 @@ Whilst Auth0 supports numerous workflows, web based workflows using Auth0 [Unive
...
+* How do I provision the organization itself in my system?
+<% } %>
 * Can I use Auth0 as an identity store?
 * Can I use my own (legacy) identity store with Auth0?
 * Can I migrate user identities from my identity store to Auth0?
+<% if (platform==="b2c") { %>
 * Can my users sign up using their existing social - e.g. Facebook or Google - accounts?
+<% } %>
+<% if (platform==="b2b") { %>
+* Can your users sign up using their organization's identity provider?
```
Comment (mostekcm) [inline articles/architecture-scenarios/_includes/_provisioning/_introduction.md:21]:
```suggestion
* Can my users sign up using their organization's identity provider?
```

File: articles/architecture-scenarios/_includes/_provisioning/_introduction.md [hunk 14]
```diff
@@ -6,10 +6,23 @@ Whilst Auth0 supports numerous workflows, web based workflows using Auth0 [Unive
...
+* Can your users be invited or self register?
```
Comment (mostekcm) [inline articles/architecture-scenarios/_includes/_provisioning/_introduction.md:22]:
```suggestion
* Can my users be invited or self register?
```

File: articles/architecture-scenarios/implementation/b2b/b2b-authentication.md [hunk 15]
```diff
@@ -0,0 +1,133 @@
...
+    - universal-login
+contentType: concept
+useCase:
+    - authentication
+---
+# Authentication
+
+<%= include('../../_includes/_authentication/_introduction.md', { platform: 'b2b' }) %>
+
+Often companies need to segregate their users by organization and sometimes users can have access to more than one organization.  Knowing which of these scenarios is relevant to your company will help define how to determine in which connection a user exists: whether you need to do it, when you need to do it, and how to accomplish it.  See [Home Realm Discovery](#home-realm-discovery) to determine if this is something relevant to your company.
```
Comment (mostekcm) [inline articles/architecture-scenarios/implementation/b2b/b2b-authentication.md]: We should eventually move this to the _introduction include to be consistent, but that seems too painful to do right now.

File: articles/architecture-scenarios/implementation/b2b/b2b-authentication.md [hunk 16]
```diff
@@ -0,0 +1,133 @@
...
+
+::: panel Best Practice
+Once you start supporting enterprise connections for users, you must do some form of [Home Realm Discovery](#home-realm-discovery) so that you can determine which connection to send the user to for authentication.
+:::
+
+With enterprise connection support, user identities and credentials are managed by the identity provider of your customers' organization, as well as certain identity claims - which Auth0 will use to populate the user [profile](/architecture-scenarios/implementation/b2b/b2b-profile-mgmt).
+
+::: panel Best Practice
+"Bring your own identity" is a great feature to provide, but if you don't support this from day one, and sometimes even if you do, you may have an organization that wants to switch to their own IDP after already having used the application for a while.  You will need a way to [Link User Accounts](https://auth0.com/docs/link-accounts) (a.k.a. Account Linking) to provide an effective way of associating the new identity with the old database identity.
```
Comment (mostekcm) [inline articles/architecture-scenarios/implementation/b2b/b2b-authentication.md]:
```suggestion
"Bring your own identity" is a great feature to provide, but if you don't support this from day one, and sometimes even if you do, you may have an organization that wants to switch to their own IDP after already having used the application for a while.  You will need a way to [Link User Accounts](/link-accounts) (a.k.a. Account Linking) to provide an effective way of associating the new identity with the old database identity.
```

File: articles/architecture-scenarios/_includes/_authentication/_introduction.md [hunk 17]
```diff
@@ -1,28 +1,38 @@
...
 
 Allowing users to sign in via user ID/password credentials means that you're not reliant on the status of third-party identity providers for your users to access your system. You also have the means require the credentials used to align with your corporate policies. Auth0 assists with this by providing you with multiple options in support of user ID/password logins, and the [guidance provided](#username-and-password-authentication) will help you understand you can leverage these options. Adding [social](#social-authentication) support at some stage, as an additional primary authentication factor, gives you added flexibility and can help you better understand your users without the need to question them further by leveraging the information already stored by the various social login [providers](https://auth0.com/docs/identityproviders#social).
```
Comment (mostekcm) [inline articles/architecture-scenarios/_includes/_authentication/_introduction.md:32]:
```suggestion
Allowing users to sign in via user ID/password credentials means that you're not reliant on the status of third-party identity providers for your users to access your system. You also have the means require the credentials used to align with your corporate policies. Auth0 assists with this by providing you with multiple options in support of user ID/password logins, and the [guidance provided](#username-and-password-authentication) will help you understand you can leverage these options. Adding [social](#social-authentication) support at some stage, as an additional primary authentication factor, gives you added flexibility and can help you better understand your users without the need to question them further by leveraging the information already stored by the various social login [providers](/identityproviders#social).
```

File: articles/architecture-scenarios/_includes/_authentication/_m2m.md [hunk 18]
```diff
@@ -0,0 +1,14 @@
...
+::: panel best practice
+Traditionally, a special "service account" would have been created in order to cater for these scenarios: a user with a username and password that was configured for services which supported non-interactive use cases. That is no longer a recommended approach for many reasons, and the current best practice is to use [OAuth 2.0 Client Credentials Grant](https://auth0.com/docs/flows/concepts/client-credentials) in these situations.
```
Comment (mostekcm) [inline articles/architecture-scenarios/_includes/_authentication/_m2m.md]:
```suggestion
Traditionally, a special "service account" would have been created in order to cater for these scenarios: a user with a username and password that was configured for services which supported non-interactive use cases. That is no longer a recommended approach for many reasons, and the current best practice is to use [OAuth 2.0 Client Credentials Grant](/flows/concepts/client-credentials) in these situations.
```

File: articles/architecture-scenarios/_includes/_authorization/_application-integration.md [hunk 19]
```diff
@@ -8,6 +8,10 @@ Using Rule extensibility, Auth0 allows you to easily [add custom claims to an ID
...
 :::
 
+<% if (platform === "b2b") { %>
+If you are creating different instances of your application for your customer organizations, a common practice is to create a custom claim in your ID token to represent the user's organization. For example, `context.idToken["http://yourdomain.com/claims/organization"]= "organization A";`
+<%  } %>
+
 ### ID Token scopes
 
 [OIDC Scopes](/scopes/current/oidc-scopes) are typically used by an application to obtain consent for authorized access to a user's details during authentication. Each of the pre-defined scopes returns the set of [standard claims](/scopes/current/oidc-scopes#standard-claims) where defined, and as described in the [OIDC specification](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims). The scopes an application requests depend on which user attributes the application needs. Once the requested scopes are authorized by the user, the claims are returned in the ID Token and are also made available via the [/userinfo](https://auth0.com/docs/api/authentication#get-user-info) endpoint.
```
Comment (mostekcm) [inline articles/architecture-scenarios/_includes/_authorization/_application-integration.md:17]:
```suggestion
[OIDC Scopes](/scopes/current/oidc-scopes) are typically used by an application to obtain consent for authorized access to a user's details during authentication. Each of the pre-defined scopes returns the set of [standard claims](/scopes/current/oidc-scopes#standard-claims) where defined, and as described in the [OIDC specification](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims). The scopes an application requests depend on which user attributes the application needs. Once the requested scopes are authorized by the user, the claims are returned in the ID Token and are also made available via the [/userinfo](/api/authentication#get-user-info) endpoint.
```

File: articles/architecture-scenarios/_includes/_authorization/_m2m.md [hunk 20]
```diff
@@ -0,0 +1,13 @@
...
+* A cron job or other service that needs to communicate with your API (e.g. where a daily report needs to be generated and emailed it to an administrator).
+* A separate API the supports privileged access (e.g. the API is not exposed to users directly, but instead to a backend only).
+* In certain microservice architectures, where some API layers need to communicate to other API layers without a user involvement, or after a user token has expired.
+* A privileged API that may need to be called before a user has authenticated (i.e. from a rule or custom DB script in your Auth0 tenant)
+
+::: panel best practice
+Traditionally, a special "service account" would have been created in order to cater for these scenarios: a user with a username and password that was configured for services which supported non-interactive use cases. That is no longer a recommended approach for many reasons, and the current best practice is to use [OAuth 2.0 Client Credentials Grant](https://auth0.com/docs/flows/concepts/client-credentials) in these situations.
```
Comment (mostekcm) [inline articles/architecture-scenarios/_includes/_authorization/_m2m.md]:
```suggestion
Traditionally, a special "service account" would have been created in order to cater for these scenarios: a user with a username and password that was configured for services which supported non-interactive use cases. That is no longer a recommended approach for many reasons, and the current best practice is to use [OAuth 2.0 Client Credentials Grant](/flows/concepts/client-credentials) in these situations.
```

File: articles/architecture-scenarios/implementation/b2b/b2b-authentication.md [hunk 21]
```diff
@@ -0,0 +1,133 @@
...
+
+::: warning
+This method will only work if you are not sharing users between organizations.  If you are sharing users within organizations, then you must support [Home Realm Discovery on the Universal Login Page](#hrd-through-universal-login)
+:::
+
+::: panel Best practice
+If your application already knows what connection (IDP) the user needs, then pass that along when you redirect the user to /authorize using the connection query parameter.
+:::
+
+If this is the case for your application(s) then home realm discovery is a simple matter of storing the Auth0 connection name with the organization specific application configuration and sending that connection name as a parameter when redirecting the user for Universal Login. Sending the connection parameter can be achieved by adding it as a query parameter when you redirect them to the authorize endpoint. For more information see the [Authentication API docs](https://auth0.com/docs/api/authentication#authorization-code-flow); however, you will generally accomplish this using the SDK for whichever language your application is written in.
```
Comment (mostekcm) [inline articles/architecture-scenarios/implementation/b2b/b2b-authentication.md]:
```suggestion
If this is the case for your application(s) then home realm discovery is a simple matter of storing the Auth0 connection name with the organization specific application configuration and sending that connection name as a parameter when redirecting the user for Universal Login. Sending the connection parameter can be achieved by adding it as a query parameter when you redirect them to the authorize endpoint. For more information see the [Authentication API docs](/api/authentication#authorization-code-flow); however, you will generally accomplish this using the SDK for whichever language your application is written in.
```

File: articles/architecture-scenarios/implementation/b2b/b2b-authentication.md [hunk 22]
```diff
@@ -0,0 +1,133 @@
...
+
+<%= include('../../_includes/_authentication/_application-integration.md', { platform: 'b2b' }) %>
+
+## Anomaly detection
+
+<%= include('../../_includes/_authentication/_anomaly-detection.md', { platform: 'b2b' }) %>
+
+## Enterprise Login
+
+The “bring your own identity” scenario has become a must-have for almost all B2B applications.  Most enterprise companies expect to be able to integrate their IDP into your application so their employees don't need to store another set of credentials.  This is a valuable way of simplifying the user authentication experience without compromising security, and using [Universal Login](#universal-login) makes it easy to start adding support for [Enterprise Connections](https://auth0.com/docs/identityproviders#enterprise) with minimal disruption.
```
Comment (mostekcm) [inline articles/architecture-scenarios/implementation/b2b/b2b-authentication.md]:
```suggestion
The “bring your own identity” scenario has become a must-have for almost all B2B applications.  Most enterprise companies expect to be able to integrate their IDP into your application so their employees don't need to store another set of credentials.  This is a valuable way of simplifying the user authentication experience without compromising security, and using [Universal Login](#universal-login) makes it easy to start adding support for [Enterprise Connections](/identityproviders#enterprise) with minimal disruption.
```

File: articles/architecture-scenarios/implementation/b2b/b2b-branding.md [hunk 23]
```diff
@@ -0,0 +1,69 @@
...
+
+<%= include('../../_includes/_branding/_introduction.md', { platform: 'b2b' }) %>
+
+## Universal login and login pages
+
+<%= include('../../_includes/_branding/_universal-login.md', { platform: 'b2b' }) %>
+
+## Branding login by organization
+
+Whether or not you need to do special customization on the Universal Login page is determined by how you plan to manage your customers’ organizations.  Before reading through this section, make sure you have read through the [Universal Login section](#universal-login-and-login-pages) and know how you are approaching organizations by reviewing [Multiple Organization Architecture](https://drive.google.com/a/auth0.com/file/d/1y2G8RNHTBujcCrnMRhp6_phQiRAkZzfF/view?usp=sharing).
```
Comment (mostekcm) [inline articles/architecture-scenarios/implementation/b2b/b2b-branding.md]:
```suggestion
Whether or not you need to do special customization on the Universal Login page is determined by how you plan to manage your customers’ organization.  Before reading through this section, make sure you have read through the [Universal Login section](#universal-login-and-login-pages) and know how you are approaching organizations by reviewing [Multiple Organization Architecture](https://drive.google.com/a/auth0.com/file/d/1y2G8RNHTBujcCrnMRhp6_phQiRAkZzfF/view?usp=sharing).
```

File: articles/architecture-scenarios/implementation/b2b/b2b-provisioning.md [hunk 24]
```diff
@@ -0,0 +1,131 @@
...
+  - store-user-data
+---
+# Provisioning
+
+<%= include('../../_includes/_provisioning/_introduction.md', { platform: 'b2b' }) %>
+
+## Provisioning organizations
+
+::: panel best practice
+What you need to do when provisioning an organization will depend on how organizations are represented in your system.  This can take some time to step back and consider how users of those organizations will be interacting with your applications.  Please see our [Multiple Organization Architecture documents](https://drive.google.com/a/auth0.com/file/d/1y2G8RNHTBujcCrnMRhp6_phQiRAkZzfF/view?usp=sharing) to get a better feel for how to decide how to have organizations represented in your IAM system.
```
Comment (mostekcm) [inline articles/architecture-scenarios/implementation/b2b/b2b-provisioning.md]: We should move this document too the docs repo too.

**Review by mostekcm — CHANGES_REQUESTED** [review]
Couple of changes

File: articles/architecture-scenarios/implementation/b2b/b2b-authentication.md [hunk 25]
```diff
@@ -0,0 +1,133 @@
...
+
+<%= include('../../_includes/_authentication/_m2m.md', { platform: 'b2b' }) %>
+
+## Multi-factor authentication (MFA)
+
+<%= include('../../_includes/_authentication/_mfa.md', { platform: 'b2b' }) %>
+
+## Planning
+
+<%= include('../../_includes/_planning.md'), { platform: 'b2b' }) %>
```
Comment (mostekcm) [inline articles/architecture-scenarios/implementation/b2b/b2b-authentication.md]:
```suggestion
<%= include('../../_includes/_planning.md', { platform: 'b2b' }) %>
```

File: articles/architecture-scenarios/_includes/_base-ways-to-integrate.md [hunk 26]
```diff
@@ -0,0 +1,3 @@
+There are many different ways Auth0 can be integrated into the <% if (platform === "b2c") { %>CIAM<% } else { %>B2B IAM<% } %> project architecture. Auth0's flexibility comprehensively supports many different use cases, however keep in mind that not every project requires 100% of the capabilities provided by Auth0.
+
+When you embark on your journey to integrate with Auth0, there are many things for you to consider. Knowing what, when, and how best to implement something will help you focus on completing the necessary tasks at the right time. To help you with this, we've put together some [planning guidance](https://drive.google.com/a/auth0.com/file/d/1lQU-uPTfwEi58WJMKL2azUhZhvy9h1IU/view?usp=sharing) that details our recommended strategies.
```
Comment (mostekcm) [inline articles/architecture-scenarios/_includes/_base-ways-to-integrate.md]:
```suggestion
When you embark on your journey to integrate with Auth0, there are many things for you to consider. Knowing what, when, and how best to implement something will help you focus on completing the necessary tasks at the right time.  

<%= include('./_planning.md', { platform: platform }) %>
```

File: articles/architecture-scenarios/_includes/_authentication/_introduction.md (hunk 2 above)
Comment (tamigoodall) [reply articles/architecture-scenarios/_includes/_authentication/_introduction.md:36]: No - I'll add glossary stuff later

File: articles/architecture-scenarios/_includes/_provisioning/_introduction.md (hunk 13 above)
Comment (tamigoodall) [reply articles/architecture-scenarios/_includes/_provisioning/_introduction.md:21]: Nope

File: articles/architecture-scenarios/_includes/_provisioning/_introduction.md (hunk 14 above)
Comment (tamigoodall) [reply articles/architecture-scenarios/_includes/_provisioning/_introduction.md:22]: Nope

File: articles/architecture-scenarios/implementation/b2b/b2b-authentication.md (hunk 15 above)
Comment (tamigoodall) [reply articles/architecture-scenarios/implementation/b2b/b2b-authentication.md]: Done

File: articles/architecture-scenarios/implementation/b2b/b2b-provisioning.md (hunk 24 above)
Comment (tamigoodall) [reply articles/architecture-scenarios/implementation/b2b/b2b-provisioning.md]: Done

File: articles/architecture-scenarios/implementation/b2c/b2c-authentication.md (hunk 7 above)
Comment (tamigoodall) [reply articles/architecture-scenarios/implementation/b2c/b2c-authentication.md]: Done

File: articles/architecture-scenarios/implementation/b2c/b2c-authentication.md (hunk 8 above)
Comment (tamigoodall) [reply articles/architecture-scenarios/implementation/b2c/b2c-authentication.md]: Done

File: articles/architecture-scenarios/implementation/b2c/b2c-authentication.md (hunk 9 above)
Comment (tamigoodall) [reply articles/architecture-scenarios/implementation/b2c/b2c-authentication.md]: Done

File: articles/architecture-scenarios/implementation/b2c/b2c-authentication.md (hunk 10 above)
Comment (tamigoodall) [reply articles/architecture-scenarios/implementation/b2c/b2c-authentication.md]: Done

File: articles/architecture-scenarios/_includes/_authentication/_introduction.md (hunk 1 above)
Comment (tamigoodall) [reply articles/architecture-scenarios/_includes/_authentication/_introduction.md]: I'll add the glossary popups later

File: articles/architecture-scenarios/implementation/b2b/b2b-authentication.md (hunk 3 above)
Comment (tamigoodall) [reply articles/architecture-scenarios/implementation/b2b/b2b-authentication.md]: Done

File: articles/architecture-scenarios/implementation/b2b/b2b-authentication.md (hunk 5 above)
Comment (tamigoodall) [reply articles/architecture-scenarios/implementation/b2b/b2b-authentication.md]: Done

File: articles/architecture-scenarios/_includes/_authentication/_m2m.md (hunk 11 above)
Comment (tamigoodall) [reply articles/architecture-scenarios/_includes/_authentication/_m2m.md]: Done

**Review by mostekcm — APPROVED** [review]

**Pull request merged**